Exercise/compute_lowest_terms.py

```diff
@@ -1,4 +1,33 @@
 ## TODO: complete the function "lowest_terms" below
-
+##lowest terms
 def lowest_terms(x):
-	return ""
+	""" split a string into a list after the / operator """
+	x = x.split('/')
+	numerator = int(x[0])
+	denominator = int(x[1])
+
+	
+	if denominator == 0:
+		return "Undefined" 
+	elif numerator == 0:
+		return "0"
+	else:
+		return reduceFrac(numerator, denominator)
+	
+
+""" using recursion to get the greatest common divisor 
+	keeping it simple with Lambda
+
+	this collects integer a and integer b and returns their greatest common divisor
+"""
```

Comment:
do not use docstrings anywhere else except at the beginning of a function. You can use # for other comments.